doc/release_notes/release_notes_2.5.rst

```diff
@@ -57,93 +57,67 @@ Upgrading to v2.5 From Previous Releases
 We highly recommended that you follow the instructions below to
 upgrade to v2.5 from previous ACRN releases.
 
-Additional Dependencies
-=======================
+Generate New Board XML
+======================
 
-Python version 3.6 or higher is required to build ACRN v2.5. You can check the version of
-Python you are using by:
+Board XML files, generated by ACRN board inspector, contain board information
+that is essential to build ACRN. Compared to previous versions, ACRN v2.5
+extends the schema of board XMLs to summarize board information more
+systematically. It is thus required to regenerate the board XML using the new
+board inspector when you upgrade to ACRN v2.5.
 
-.. code-block:: bash
-
-   $ python3 --version
-   Python 3.5.2
-
-Only when the reported version is less than 3.6 (as is the case in the example above) do
-you need an upgrade. The first (and preferred) choice is to install the latest
-Python 3 from the official package repository:
+To use the new board inspector, ensure you have Python >= 3.6 on the target
+board with the package lxml installed. Refer to :ref:`upgrade_python` for
+detailed steps to check and upgrade Python version. The lxml package can be
+installed by executing the following command:
 
 .. code-block:: bash
 
-   $ sudo apt install python3
-   ...
-   $ python --version
-   Python 3.8.8
+   $ sudo pip3 install lxml
 
-If this does not get you an appropriate version, you may use the deadsnakes PPA
-(using the instructions below) or build from source yourself.
+.. note::
+   This is not the complete list of tools required to run the board
+   inspector. Refer to :ref:`acrn_config_workflow` for a complete guide.
 
-.. code-block:: bash
+With the prerequisites, copying the board inspector (avaliable at
```

Comment:
There are lots of files in the board_inspector folder.  Do we need to copy the whole folder to the target?
Also should we tell the user to set a value for the BOARD variable so the instructions work as written, or tweak the instructions to say ``my_board_name`` or something?

Assuming yes on copying the entire folder, change to:
```
With the prerequisites ready, copy the entire board inspector folder from
``misc/config_tools/board_inspector`` to the target board, ``cd`` to that directory
on the target, and run the tool using::

   sudo python3 cli.py $(BOARD) 

This will generate a ``$(BOARD).xml`` in the current working directory.
```

I doesn't seem intuitive that the board inspector tool is called ``cli.py`` instead of something like ``board_inspector.py``.

Reply:
Yes, the whole folder needs to be copied. Will update the instructions in v2.

As for the name of ``cli.py``, Geoffroy and I had some discussions at https://github.com/projectacrn/acrn-hypervisor/pull/6213#issuecomment-862125944